README.md

```diff
@@ -184,27 +88,25 @@ These lines are based on tests written using `rspec`.
 * First, create a private key. This key should be created once per test suite for performance matters.
 ```ruby
 config.before(:suite) do
-  $private_key = OpenSSL::PKey::RSA.generate(1024)
+  $private_key = OpenSSL::PKey::RSA.generate(2048)
 end
 ```
 * Then, in a `shared_context`, configure a lazy token based on your main user. (here, we assume you have a `user` variable with a `keycloak_id` property)
 ```ruby
 let(:jwt) do
   claims = {
-    iat: Time.zone.now.to_i,
-    exp: (Time.zone.now + 1.day).to_i,
```

Comment:
Just double-check that not having the time zone specific time has any affect on anything.